distributed/tests/test_security.py

```diff
@@ -104,24 +110,22 @@ def test_kwargs():
 
 
 def test_repr():
-    with new_config({}):
-        sec = Security(tls_ca_file="ca.pem", tls_scheduler_cert="scert.pem")
-        assert (
-            repr(sec)
-            == "Security(tls_ca_file='ca.pem', tls_scheduler_cert='scert.pem')"
-        )
+    sec = Security(tls_ca_file="ca.pem", tls_scheduler_cert="scert.pem")
+    assert (
+        repr(sec)
+        == "Security(require_encryption=False, tls_ca_file='ca.pem', tls_scheduler_cert='scert.pem')"
```

Comment:
Super-minor nit (I wouldn't bother changing this at this point).  I tend to avoid rigid checks like this.  I might instead test a few things like `"scert.pem" in repr(sec)`.  We don't really